blog/2015/12/software-carpentry-workshop-eitn-paris.html

```diff
@@ -0,0 +1,48 @@
+</p>
+
+<p>
+The course was closed by a tutorial by Bartosz Teleńczuk, who re-visited programming in Python language and presented how to work with structured data using a popular Python library called pandas replacing the standard SQL lesson of SC syllabus. As a continuation of the morning lecture he started with a demonstration how to contribute to existing open-source projects, such as pandas or numpy, on Github. The participants could work on their version of the materials he shared on GitHub and solve the included exercises. He used an interactive environment called IPython notebook, which minimised the problems with the initial setup while allowing to responding interactively to participants' questions. Apart from the basics of pandas, he covered Python's dictionaries and various ways of array indexing. Again the limited time did not allow to discuss more advanced topics, but the students were offered some starting pointers for self-study.
+</p>
+
+<p>
+For most instructors and helpers this was the first experience with a Software Carpentry workshop. Everyone was amazed with the quality of the SC materials  and students liked the steady flow of the lessons. Many students enjoyed the practice-oriented introduction to programming in Python, but some would prefer a more classical overview of the language syntax. The students did not use the post-it notes as instructed &mdash; they either didn't use them at all or kept the green one sticked to their laptops at all times (we used green for completed task and red to signal a problem/question). Perhaps next time we could instruct helpers to encourage students to use them more. The logistics worked quite well thanks to a good local team (4 people: one for the website and coordination between speakers and helpers; one resposible for finances and venue; one taking care of the registarations, bookings and coffee breaks; and one for IT support). On the first day we had nearly one hour of delay because the speakers lunch draged out much past the planned 1.5h slot, which was bad for the general atmosphere of the afternoon session. The programming experience of the students was quite broadly distributed &mdash; from no experience at all to experienced programmers, which made it difficult to adjust the level. From the initial feedback we learnt that the novice programmers enjoyed the course a lot, but for advanced it might have been too slow and too shallow (we didn't get much negative feedback, but this is probably due to the response bias). We didn't ask for interim feedback during the classes, but next time it might be better to ask for feedback before each coffee break. 
```

Comment:
There are a few references in here like “everyone managed to catch up”, “everyone was comfortable”, and this “Everyone was amazed“.  It's possible that you just had a very uniform class, but if so I'd be careful to back those sorts of claims up ;).  Otherwise I'd relax the claims a bit to allow for the possibility that there were a few stragglers and sceptics.  Listing reported learner likes and dislikes (like you do in the rest of this paragraph) is safer and presents a more detailed view.